bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/target/env/GuideProbeTargets.java

```diff
@@ -366,6 +366,11 @@ public boolean primaryIsBagsTarget() {
     }
 
     /**
+     * Determine if the primary guide star exists but is not a BAGS target.
+     */
+    public boolean primaryIsManual() { return primaryTarget.exists(manualTargets::contains); }
```

Comment:
It seems like it there wouldn't be an issue with having a bags result that happens to be one of the manual options?  Could you flip the test?  That is,

primary is bags = primary exists and is equal to the bags result target (if any)
primary is manual = ! primary is bags

In fact, if I recall the fact that there is a bags result means it must be primary?